src/main/groovy/jenkins/automation/utils/GhUtils.groovy

```diff
@@ -12,51 +14,87 @@ class GhUtils {
  * This is a wrapper for the <a href="https://wiki.jenkins.io/display/JENKINS/GitHub+pull+request+builder+plugin">GitHub Pull Request Builder Plugin</a>.
  * @see <a href="https://github.com/cfpb/jenkins-automation/blob/gh-pages/docs/examples.md#using-gh-pr-watcher" target="_blank">example</a>
  * @param  context delegate
- * @param  ghProject A reference to the GHE project as 'username/reponame'
- * @param  ghHostname The GitHub hostname, i.e. github.org.tld, or github.com
- * @param  ghAuthId the Jenkins credential ID for GitHub authentication
- *         see <a href="https://github.com/jenkinsci/ghprb-plugin/blob/master/README.md#credentials">the plugin docs</a>
- * @param  ghPermitAll  optional Whether or not to allow all contributors to build a PR
- * @param  ghPrHooks  optional Whether or not to install webhooks in the remote repo
- * @param  ghPrCron  optional For use when ghPrHooks is false. Uses standard cron syntax
- * @param  ghPrOrgList  optional A string of organizations to whitelist for PRs
+ * @param  ghPrConfig A <code>Map</code> of configuration for the PR builder.
+                      Valid keys:
+ * <br/>&#x2014;  <code>ghProject</code> The GitHub project name, e.g. username/reponame
```

Comment:
What are all these extra chars and markup. I prefer that these groovydoc comments be also readable in code.

Reply:
These are bullet points when rendered in HTML, and they're consistent with https://github.com/cfpb/jenkins-automation/blob/master/src/main/groovy/jenkins/automation/utils/CheckmarxUtils.groovy where I borrowed them. @imuchnik  are you objecting to all markup or just these bullets?

Reply:
Just the bullets. I think 
```
@param paramName paramDescription
```
should be sufficient.